setup.py

```diff
@@ -59,7 +59,16 @@ def _get_dbt_core_version():
     "thrift>=0.11.0,<0.17.0",
 ]
 session_extras = ["pyspark>=3.0.0,<4.0.0"]
-all_extras = odbc_extras + pyhive_extras + session_extras
+connect_extras = [
+    "pyspark==3.5.0",
```

Comment:
Can we support `pyspark>=3.4.0,<4`, or at least `pyspark>=3.5.0,<4`?

Reply:
`pyspark>=3.5.0,<4` added.
3.4.0 has an issue where temporary views are not shared between queries. If one dbt query creates a temp view, another query cannot see it. Can't find a spark issue # now